### DISCUSSION TITLE
PHP 8.2?

### DISCUSSION THREAD

**Discussion by jhedstrom** [docksal/docksal#1766] [General]
I see there is a `docksal/cli:php8.2-edge` image available, but I was wondering when will the official PHP 8.2 image be available?

**Comment by lmakarov** [comment]
On it's way (https://github.com/docksal/service-cli/pull/298). Over the weekend or early next week (once GitHub Actions build issues are resolved).

**Update**

Latest [release](https://github.com/docksal/service-cli/releases/tag/v3.4.0), with PHP 8.2:

```
docksal/cli:php8.0-3.4
docksal/cli:php8.1-3.4
docksal/cli:php8.2-3.4
```

**Reply by jhedstrom** [reply]
Awesome, thanks!
